### PR TITLE
24/Add icon fonts

### LINKED ISSUE #24: VV/EE Create and apply icon font seAp
## Plan

- [x] Generate icon fonts similar to those for Haven and Hospice Hope but relevant for seAp using Icomoon
- [x] Add icons
- [x] Acknowledge source of icons with comments in icons stylesheet

## Acceptance Criteria

- [x] icons exist and are used
- [x] @roseb04 to check icons chosen and approve their use
- [x] [dev] source(s) is/are acknowledged

### PR DESCRIPTION
**Pull request by AlinChindea**
Fixes #24

## Description

- Add icon fonts
- Apply icons

@neontribe/ca-seap

### CODE REVIEW

File: src/styles/icons.css [hunk 1]
```diff
@@ -16,60 +28,36 @@
...
+  content: "\e906";
 }
-.icon-users:before {
-  content: "\f0c0";
+.icon-info:before {
+  content: "\e907";
 }
-.icon-envelope:before {
-  content: "\f0e0";
+.icon-envelop:before {
```
Comment (katjam) [inline src/styles/icons.css]: Is the mis-spell delibrate?

File: src/Copy/BrandCopy.elm [hunk 2]
```diff
@@ -81,7 +81,7 @@ brandCopy key =
             CallToAction
                 { action = DesktopInteractive
                 , category = "email"
-                , icon = "✨"
+                , icon = "email"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm:84]: Do you mean envelop? Or you have included 2 alternatives?

**Review by katjam — COMMENTED** [review]
Looks ok but clear up the email vs envelop

File: src/Copy/BrandCopy.elm (hunk 2 above)
Comment (AlinChindea) [reply src/Copy/BrandCopy.elm:84]: 2 alternatives. I will remove the one unused.

File: src/styles/icons.css (hunk 1 above)
Comment (AlinChindea) [reply src/styles/icons.css]: yes, that's how the file was generated. there are two options: one is icon-email the other icon-envelop. I remove the latter as i did not use it.

**Review by katjam — APPROVED** [review]
Code looks good. :pager:

**Pull request merged**